e2e/test/scenarios/embedding/embedding-linked-filters.cy.spec.js

```diff
@@ -86,7 +86,7 @@ describe("scenarios > embedding > dashboard > linked filters (metabase#13639, me
       chartPathWithFillColor("#509EE3").should("have.length", 1).realHover();
       assertEChartsTooltip({
         header: "AK",
-        rows: [{ color: "#509EE3", name: "Count", value: "68" }],
```

Comment:
We capitalized _some_ column display names on FE, also those coming from native queries. I've decided to remove capitalization of just _some_ display names of native queries. User is able to pick a name they like capitalized or not. Imho leaving in capitalization of just _some_ names makes the codebase more convoluted. If you find this change not desired let's discuss that. The comment applies to the whole of this spec file.